test/jest-setup.ts

```diff
@@ -18,7 +18,7 @@ let server: SetupServer;
  */
 export const getFailOnConsoleDefaultConfig = (): failOnConsole.InitOptions => ({
 	shouldFailOnError: true,
-	shouldFailOnWarn: true
+	shouldFailOnWarn: false
```

Comment:
This will impact all the modules that use Commons as submodule. 
If this configuration is necessary it should be overridden in the Contacts `jest-setup.ts `